### DISCUSSION TITLE
Support default labels for indivudal accounts

### DISCUSSION THREAD

**Discussion by DerYeger** [community/community#10537] [Other Feature Feedback, Questions, & Ideas]
Currently, it's only possible to [manage default labels for organization accounts](https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization).

For people like me who host multiple OSS projects on their personal accounts, it would be a tremendous improvement if we could similarly manage the default labels for our repositories in a central place.

**Comment by l-zeuch** [comment]
I'd really like to see this being implemented as well -- myself I work a lot with labels, too, and setting up my "default scaffold" takes some time.

**Comment by Ernest314** [comment]
Here's a link to previous discussion on the topic (the last comment after the issue was closed is talking about the same issue): https://github.com/isaacs/github/issues/281

**Comment by TwiN** [comment]
I would definitely love to see this being implemented as well.

I'm considering creating a GHA workflow for this using something like https://github.com/marketplace/actions/label-syncer to configure the labels and https://github.com/marketplace/actions/repo-list-generator to retrieve all my repositories, but to be honest, I feel like this is a hacky way of doing something that should be directly enabled by GitHub instead (and it _is_, but only for organizations).
